src/main/java/org/cactoos/text/FormattedText.java

```diff
@@ -104,7 +104,7 @@ public FormattedText(final CharSequence ptn, final Collection<?> arguments) {
      * @param ptn Pattern
      * @param arguments Arguments
      */
-    public FormattedText(final Text ptn, final Collection<?> arguments) {
+    public FormattedText(final Text ptn, final Collection<Object> arguments) {
```

Comment:
@DmitryBarskov please keep `?`: it does not have the same meaning from the caller point of view

Reply:
@victornoel using `?` causes a compilation error. It seems that `FormattedText(final Text ptn, final Collection<?> arguments)` won't be ever invoked if we use `<?>`
```java
class FormattedText {
  FormattedText(final Text ptn, final Object... arguments) {}
  FormattedText(final Text ptn, final Collection<?> arguments) {}
}
```

Reply:
@DmitryBarskov apparently it was working before, I?m not sure why you are getting a compilation error…

Reply:
It was `Collection<Object>` before

Reply:
@DmitryBarskov ah but I see it's back to `?` and it seems to compile? or is it github that is trolling me with incorrect diffs?

Reply:
@victornoel if you click on a notification it shows changes from new commits you haven't viewed yet.
In the last one I changed it to
```java
public FormattedText(final Text ptn, final Collection<Object> arguments) {
```
https://github.com/yegor256/cactoos/pull/1620/files#diff-43305e7df83468e8164324dd7558eb57f849d632c483591f1976d92c0817f819R107

Reply:
Yes I know, but I just looked at the global diff between your MR and the master branch, and github is showing the change ^^

![image](https://user-images.githubusercontent.com/160975/126354757-5011cf36-b4bf-4cf8-8b4d-28e2a1762fc8.png)

Reply:
@victornoel yes, that's right, I reverted back the ctor on line 107.

`FormattedText(Text, Collection<Object>)`